test/Microsoft.AspNet.Mvc.FunctionalTests/MvcTestFixture.cs

```diff
@@ -67,7 +67,6 @@ public MvcTestFixture(object startupInstance)
             using (new CultureReplacer())
             {
                 _server = TestServer.Create(
-                    CallContextServiceLocator.Locator.ServiceProvider,
```

Comment:
We use this provider two other places in this fixture. Is changing just this line enough to get the functional tests working again?